datanode/entities/ledgerentry_filter.go

```diff
@@ -27,20 +27,12 @@ type LedgerEntryFilter struct {
 	TransferTypes []LedgerMovementType
 }
 
-func LedgerEntryFilterFromProto(pbFilter *v2.LedgerEntryFilter) (*LedgerEntryFilter, error) {
-	filter := LedgerEntryFilter{}
+func LedgerEntryFilterFromProto(pbFilter *v2.LedgerEntryFilter) *LedgerEntryFilter {
+	filter := &LedgerEntryFilter{}
 	if pbFilter != nil {
 		filter.CloseOnAccountFilters = CloseOnLimitOperation(pbFilter.CloseOnAccountFilters)
-
-		var err error
-		filter.FromAccountFilter, err = AccountFilterFromProto(pbFilter.FromAccountFilter)
-		if err != nil {
-			return nil, err
-		}
-		filter.ToAccountFilter, err = AccountFilterFromProto(pbFilter.ToAccountFilter)
-		if err != nil {
-			return nil, err
-		}
+		filter.FromAccountFilter = AccountFilterFromProto(pbFilter.FromAccountFilter)
```

Comment:
nit-pick: will from _and_ to account filters be set at all times? I saw the `AccountFilterFromProto` wraps things in a nil check and returns an empty filter type if the input is `nil`, but IMO relying on these conversion functions checking for nil is a bit off, especially considering we have `pbFilter != nil` in this function, and this function previously returned `nil, err` in some cases, whereas now it will always return a non-nil pointer.

Like I said: a bit of a nit-pick, but overall this gives off the impression that we're more likely to silence potential errors, maybe to the point where invalid filter values are just going to end up being ignored and the entire data-set is going to be returned unfiltered.